time-calculator/time_calculator.py

```diff
@@ -0,0 +1,67 @@
+
+def add_time(startTime, stopTime, dayOfWeek=None):
+
+    # Start Time
+    [startTime, start_am_pm] = startTime.split()
+    [startHours, startMinutes] = startTime.split(':')
+
+    # End Time
+    [stopHours, stopMinutes] = stopTime.split(":")
+
+    stopMinutes = int(stopMinutes) + (60 * int(stopHours))
+
+    if start_am_pm == "PM":
+        startHours = int(startHours) + 12
+
+    startMinutes = int(startMinutes) + (60 * int(startHours))
+    totalMinutes = startMinutes + stopMinutes
+
+    # Minutes Calculation
+    finalMinutes = totalMinutes % 60
+    finalHours = int(totalMinutes / 60)
+
+    if len(str(finalMinutes)) == 1:
+        finalMinutes = "0" + str(finalMinutes)
+    elif len(str(finalMinutes)) == 2:
+        finalMinutes = str(finalMinutes)
+
+    # Calculation Days
+    hours = finalHours % 24
+    days = int(finalHours / 24)
+
+    # Am & PM Calculation
+    finalHours = hours % 12
+
+    if int(hours / 12) == 0:
+        finalAmPm = "AM"
+        if finalHours == 0:
+            finalHours = 12
+    else:
+        finalAmPm = "PM"
```

Comment:
We are using `"PM"` as hard coded string twice .... is it better to use a common variable for it?